values.yaml

```diff
@@ -0,0 +1,19 @@
+metadataservice:
+  image:
+    repository: ghcr.io/metal-toolbox/hollow-metadataservice
+    tag: "v0.0.1"
+  env:
+    DEBUG: "true"
+  publicPaths:
+    - $
+    - metadata
+    - userdata
+    - components
```

Comment:
the metadata service currently doesn't implement any `/components` functionality. This is implemented in Kant, but appears to be unused.

Reply:
Good catch! Fixed.